jupyter-book/preamble.md

```diff
@@ -12,53 +12,57 @@ Interested in contributing? Please have a look at https://github.com/theislab/si
+The human body is a complex machine that heavily relies on the basic units of life - cells. These cells exhibit remarkable diversity, varying in type and function, and can undergo significant transitions during development, in response to disease, or during regenerative processes. This cellular heterogeneity manifests in their structure, function, and gene expression profiles. Disruptions to this delicate balance can lead to systemic dysregulation, contributing to severe conditions such as cancer {cite}`Macaulay2017`. Therefore, understanding how cells behave under normal and perturbated states is critical for improving our knowledge of the entire cellular systems.
 
-This monumental task is approached in different ways of which the most promising one is to profile cells at the individual level. So far, each cells' transcriptome was primarily examined in a process known as single-cell RNA sequencing. With recent advances in single-cell genomics, it is now possible to enrich the transcriptome information with spatial, chromatin accessibility or protein information. These advances generate not only insight into complex regulatory mechanisms, but also go along with additional complexity for data analysts.
+To address this challenge, researchers employ a variety of strategies, with one of the most promising being profiling cells at the individual level. Traditionally, each cells' transcriptome was primarily examined in a process known as single-cell RNA sequencing. However, recent advancements in single-cell genomics now enable the integration of transcriptome data with spatial, chromatin accessibility, or protein-level information. These developments not only enhance our understanding of complex regulatory mechanisms but also introduce additional challenges in data analysis.
 
-Nowadays, data analysts are facing a vast analysis tool landscape with a collection of more than 1000 computational single-cell analysis methods. It is becoming increasingly challenging to navigate this large range of different tools to generate sound results which are at the forefront of science.
+Currently, analysts are faced with an overwhelming array of computational tools - over 1,000 methods dedicated to single-cell analysis alone. Navigating this extensive landscape to produce reliable, cutting-edge results poses a significant challenge.
```

Comment:
```suggestion
Currently, analysts are faced with an overwhelming array of computational tools - over 1,000 methods dedicated to single-cell analysis alone.
Navigating this extensive landscape to produce reliable, cutting-edge results poses a significant challenge.
```

Could you update the number, please? The source is https://www.scrna-tools.org/ Also it's over 1xxx for **single-cell RNA-Seq** alone. It's from https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02519-4 and we should consider citing it here. It might already be in the citations - if not, consider adding it.

The soruce